config/projects/chef.rb

```diff
@@ -65,6 +65,7 @@
 # Chef Release version pinning
 # override :chef, version: "12.3.0"
 # override :ohai, version: "8.3.0"
+override :libffi, version: "3.2.1" if ppc64? || ppc64le?
```

Comment:
We restricted to ppc64 only to get it done quickly. May be we have CI boxes that can verify this on other platforms or some VMs for other platforms I can test this on? Sharing with you separately a list of some other packages that can be upgraded to avoid the patches.